src/Models/EventModel.php

```
@@ -1,286 +1,303 @@
+            } else if (isset($_POST['Country']) && $_POST['Country'] == "Duitsland"){
+                if (isset($_POST['Address']) && !preg_match("/^\d{5}$/", $_POST['Address'])) {
+                    $errors[] = "De postcode moet bestaan uit 5 cijfers";
+                }
+            } else if (isset($_POST['Country']) && $_POST['Country'] == "Luxemburg"){
+                if (isset($_POST['Address']) && !preg_match("/^\d{4}$/", $_POST['Address'])) {
+                    $errors[] = "De postcode moet bestaan uit 4 cijfers";
+                }
+            }
+            if (isset($_POST['banner'])) {
+                $img = file_get_contents($_POST['banner']);
+                $data = base64_encode($img);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$data'. (undefined)

(UnusedLocalVariable)